pyro/distributions/distribution.py

```diff
@@ -88,7 +88,7 @@ class Distribution(object):
     reparameterized = False
     enumerable = False
 
-    def batch_shape(self, x=None, *args, **kwargs):
+    def batch_shape(self, *args, **kwargs):
```

Comment:
Nice! It might be nice to drop random primitive and make this an actual `property`, but that can wait.